docs/rafter/08-02-set-minio-to-gateway-mode.md

```diff
@@ -0,0 +1,409 @@
+---
+title: Set MinIO to Gateway mode
+type: Tutorials
+---
+
+
+By default, you install Kyma with Rafter in MinIO stand-alone mode. This tutorial shows how to set MinIO to Gateway mode on different cloud providers using an [override](/root/kyma/#configuration-helm-overrides-for-kyma-installation).
+
+>**TIP:** Every cloud provider offers a different payment policy for storing buckets. To avoid unexpected costs, verify the payment policy with the given provider before you start using Gateway mode. To reduce the costs in general, always try to limit the number of buckets and create them for groups and domains rather than separate assets.
+
+## Prerequisites
+
+<div tabs name="prerequisites" group="gateway-mode">
+  <details>
+  <summary label="google-cloud-storage">
+  Google Cloud Storage
+  </summary>
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [gcloud](https://cloud.google.com/sdk/gcloud/)
+- [Google Cloud Platform (GCP)](https://cloud.google.com) project
+
+  </details>
+  <details>
+  <summary label="azure-blob-storage">
+  Azure Blob Storage
+  </summary>
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure)
+- [Microsoft Azure](http://azure.com) subscription
+
+  </details>
+  <details>
+  <summary label="aws-s3">
+  AWS S3
+  </summary>
+
+>**CAUTION:** AWS S3 Gateway mode was only tested manually on Kyma 1.6. Currently, there is no automated pipeline for it in Kyma.
```

Comment:
AWS S3 Gateway mode was only tested manually on Kyma >= 1.6

Reply:
As discussed, we will keep it this way.